### PR TITLE
Add Pylons to osmosis.zone_assets.json

### PR DESCRIPTION
**Pull request by MikeSofaer**
## Description

Adding Bedrock from Pylons

### Adding Chains

If adding a new chain, please ensure the following:
- [X] Chain is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - Chain's registration must have `staking` defined, with at least one `staking_token` denom specified.
   - Chain's registration must have `fees` defined; at least one fee token has low, average, and high gas prices defined.
- [ ] Add chain to bottom of `zone_chains.json`
   - RPC and REST must not have any CORS blocking of the Osmosis domain.
   - RCP node must have WSS enabled.

### Adding Assets

- [X] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [X] Ready -- A liquidity pool has been created. The pool ID is: 1386

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2658,6 +2658,12 @@
       "categories": [
         "meme"
       ]
+    },
+    {
+      "base_denom": "ubedrock",
+      "chain_name": "pylons",
+      "path": "transfer/channel-17863/ubedrock",
+      "osmosis_unlisted": true
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2666]: Please add "osmosis_verified": false

**Comment by JeremyParish69** [comment]
Please add the chain to zone_chains

**Comment by MikeSofaer** [comment]
@JeremyParish69 Better?

**Comment by JeremyParish69** [comment]
> @JeremyParish69 Better?

Yes, thanks!

**Comment by JeremyParish69** [comment]
There are still conflicts that must be resolved.

**Comment by MikeSofaer** [comment]
@JeremyParish69 Resolved the conflict

**Comment by JeremyParish69** [comment]
Checks are failing because there's no low, average., and high gas rates defined in the Chain Registry. Can you please define fees there?
https://github.com/cosmos/chain-registry/blob/master/pylons/chain.json#L20

**Comment by JeremyParish69** [comment]
Sorry, there are more conflicts by now. Can you please resolve them again?

**Comment by MikeSofaer** [comment]
> Sorry, there are more conflicts by now. Can you please resolve them again?

Should be resolved, thank you for your help!

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -2719,6 +2719,14 @@
       "path": "transfer/channel-20100/nscr",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "base_denom": "ubedrock",
+      "chain_name": "pylons",
+      "path": "transfer/channel-17863/ubedrock",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2726]: There is a typo in the channel. Is it 17863 or 17683?

Comment (MikeSofaer) [reply osmosis-1/osmosis.zone_assets.json:2726]: Looks like 17683 in my Keplr

**Comment by JeremyParish69** [comment]
Are the endpoints OK? I don't see the checklist fully ticked
"Add chain to bottom of zone_chains.json
RPC and REST must not have any CORS blocking of the Osmosis domain.
RCP node must have WSS enabled."
If it's good, please resolve conflicts and then we can proceed.

**Comment by MikeSofaer** [comment]
> Are the endpoints OK? I don't see the checklist fully ticked "Add chain to bottom of zone_chains.json RPC and REST must not have any CORS blocking of the Osmosis domain. RCP node must have WSS enabled." If it's good, please resolve conflicts and then we can proceed.

I think so but I am confirming with NodeStake.top. now.

**Pull request merged**